src/materialized/src/bin/materialized/main.rs

```diff
@@ -359,6 +359,9 @@ pub struct Args {
     /// directory.
     #[clap(long, env = "MZ_PERSIST_CONSENSUS_URL")]
     persist_consensus_url: Option<Url>,
+    /// Postgres catalog stash connection string.
+    #[clap(long, env = "MZ_CATALOG_POSTGRES_STASH", value_name = "POSTGRES_URL")]
```

Comment:
How is this postgres URL going to look like in production? Are we going to be using any randomly-generated passwords or just TLS? I think for security reasons, the password should not be part of the URL and should not be part of the command line either. It should be read from a kubernetes secret if running in a kubernetes environment.

Reply:
I'm happy to support whatever the cloud team wants to do here, but don't have many opinions myself. Thinking through the security of the postgres url, connection, and how that gets plumbed down can happen from discussion with them.

Reply:
The cloud team can plumb the URL securely via the `MZ_CATALOG_POSTGRES_STASH` environment variable!